src/chinese_language_model.js

```diff
@@ -0,0 +1,73 @@
+import LanguageModel from './language_model.js'
+import Feature from './feature.js'
+import * as Constants from './constants.js'
+
+let typeFeatures = new Map()
+let typeFeaturesInitialized = false
+
+/**
+ * @class  LatinLanguageModel is the lass for Latin specific behavior
+ */
+export default class ChineseLanguageModel extends LanguageModel {
+  static get languageID () { return Constants.LANG_CHINESE }
+  static get languageCode () { return Constants.STR_LANG_CODE_ZHO }
+  static get languageCodes () { return [Constants.STR_LANG_CODE_ZH, Constants.STR_LANG_CODE_ZHO] }
+  static get contextForward () { return 1 }
+  static get contextBackward () { return 0 }
+  static get direction () { return Constants.LANG_DIR_LTR }
+  static get baseUnit () { return Constants.LANG_UNIT_CHAR }
+
+  static get featureValues () {
+    return new Map([
+      [
+        Feature.types.fullForm,
+        []
+      ],
+
+      [
+        Feature.types.frequency,
+        []
+      ],
+      [
+        Feature.types.pronunciation,
+        []
+      ],
+      [
+        Feature.types.dialect,
+        [
+          'simple',
+          'traditional'
+        ]
+      ],
+      [
+        Feature.types.mandarin,
+        []
+      ],
+      [
+        Feature.types.cantonese,
+        []
+      ],
+      [
+        Feature.types.tang,
+        []

```

Comment:
I know it's very complicated and I am not even distantly familiar with Chinese, but I'm wondering it that would be the best way to classify the varieties of the language.

From the [article](https://www.vengaglobal.com/blog/simplified-traditional-chinese-mandarin-cantonese/) I understood that traditional Chinese and simplified Chinese are more of writing methods (different ways to record approximately the same words) than dialects, even though now there are some terms and concepts unique to one or the other. Mandarin and Cantonese are varieties of a spoken language. So if we group simplified and traditional Chinese together maybe we shall group Mandarin and Cantonese too (should then they be dialects)? Or maybe all four belong to the same plane? @balmas, what do you think?

Reply:
so I agree let's remove Feature.types.cantonese, Feature.types.tang, Feature.types.mandarin,Feature.types.dialect.simple, and Feature.types.dialect.traditional from the data model

Reply:
oh, good catch! I am sorry - I have forgotten to remove those features!